rollup.config.js

```diff
@@ -0,0 +1,55 @@
+const rollup = require('rollup').rollup;
```

Comment:
Sure. But I'm still in doubt if I should bundle two different version, one using UMD and another CommonJS modules. Maybe just one is enough. What do you think @evgenyrodionov ?